### PR TITLE
Use Unscaled Time Internally

### PR DESCRIPTION
**Pull request by awohns**
Remove "Ne" from likelihood calculations and only use it in the prior. Doesn't affect output at all.
Look good @hyanwong ?

### CODE REVIEW

File: tsdate/date.py [hunk 1]
```diff
@@ -858,12 +858,10 @@ def outside_maximization(self, Ne, *, eps, progress=None):
...
-        return (
-            self.lik.timepoints[np.array(maximized_node_times).astype("int")] * 2 * Ne
-        )
+        return self.lik.timepoints[np.array(maximized_node_times).astype("int")]
 
 
-def posterior_mean_var(ts, timepoints, posterior, Ne, *, fixed_node_set=None):
+def posterior_mean_var(ts, timepoints, posterior, *, fixed_node_set=None):
     """
     Mean and variance of node age in scaled time. Fixed nodes will be given a mean
```
Comment (hyanwong) [inline tsdate/date.py:866]: This is not in scaled time any more is it?

**Comment by hyanwong** [comment]
Looks great to me. Do you agree that it makes the algorithm easier to understand / more logical?

**Comment by hyanwong** [comment]
We should adjust the docs for the main `date()` method to state that `Ne` is (now) only used to make the prior, so if you provide your own prior, the `Ne` parameter is ignored.

**Comment by hyanwong** [comment]
Can this be merged now?

**Comment by awohns** [comment]
Not quite yet, I need to write a few more tests then will merge. Thanks for checking in on it though, fell off my radar

**Comment by swamidass** [comment]
Is this going to be merged at any point soon?

**Comment by awohns** [comment]
Yep, will merge this week @swamidass

**Comment by swamidass** [comment]
Okay. I need this for that Ne implementation.

**Pull request merged**